cmd/masa-node/main.go

```diff
@@ -89,7 +89,7 @@ func main() {
 	// Init cache resolver
 	go db.InitResolverCache(node, keyManager)
 	// Subscribe to blocks -- moving to new ticket
```

Comment:
maybe we can drop the comment too?
```suggestion
	// Subscribe to blocks
```